vagrant/roles/local.defaults/vars/main.yml

```diff
@@ -160,3 +160,7 @@ environments:
         networks:
           public: 5
         groups: [clients]
+
+misc:
+  logs:
+    basedir: "/tmp"
```

Comment:
Just a minor comment, but I would move this upper in the configuration file. Having it here will make it less visible because the environment definitions are long.

Reply:
Done